python/ql/test/experimental/meta/ConceptsTest.qll

```diff
@@ -142,3 +142,42 @@ class HttpServerRouteSetupTest extends InlineExpectationsTest {
     )
   }
 }
+
+class HttpServerHttpResponseTest extends InlineExpectationsTest {
+  File file;
+
+  HttpServerHttpResponseTest() { this = "HttpServerHttpResponseTest: " + file }
```

Comment:
Should we restrict `file` further here? Say to Python-files within the test directory?